geode-core/src/distributedTest/java/org/apache/geode/distributed/LocatorDUnitTest.java

```diff
@@ -1314,17 +1314,21 @@ public void run() {
     vm1.invoke("waitUntilLocatorBecomesCoordinator", this::waitUntilLocatorBecomesCoordinator);
```

Comment:
I think that waitUntilLocatorBecomesCoordinator should also use view.getCreator(), not view.getCoordinator().  This is allowing the test to proceed past these checks before a locator is actually coordinator - the view merely thinks that one of the locators should _become_ the coordinator.

Reply:
I'll have a look at that one too, thx